lib/content_security_policy.rb

```diff
@@ -57,14 +58,55 @@ def build
 
   private
 
+  attr_reader :request
+
+  SCRIPT_ASSET_DIRECTORIES = [
+    # [dir, can_use_s3_cdn, can_use_cdn]
+    ['/assets/',             true, true],
+    ['/brotli_asset/',       true, true],
+    ['/extra-locales/',      false, false],
```

Comment:
`extra-locales` JS bundle specifically does not use CDN https://github.com/discourse/discourse/commit/daad2291ba420b820e2b007a9944ed9bfb1ff9a0